views/js/pciCreator/dev/likertMy/runtime/js/likertMy.js

```diff
@@ -0,0 +1,140 @@
+/*
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2016 (original work) Open Assessment Technologies SA;
+ *
+ */
+define([
+    'qtiCustomInteractionContext',
+    'taoQtiItem/portableLib/jquery_2_1_1',
+    'taoQtiItem/portableLib/OAT/util/event',
+    'likertMy/runtime/js/renderer'
+], function(qtiCustomInteractionContext, $, event, renderer){
+    'use strict';
+
+    var likertMy = {
+        id : -1,
+        getTypeIdentifier : function getTypeIdentifier(){
+            return 'likertMy';
+        },
+        /**
+         * Render the PCI : 
+         * @param {String} id
+         * @param {Node} dom
+         * @param {Object} config - json
+         */
+        initialize : function initialize(id, dom, config, assetManager){
+
+            var self = this;
+            this.id = id;
+            this.dom = dom;
+            this.config = config || {};
+
+            //add method on(), off() and trigger() to the current object
+            event.addEventMgr(this);
+
+            renderer.render(this.id, this.dom, this.config, assetManager);
+
+            //tell the rendering engine that I am ready
+            qtiCustomInteractionContext.notifyReady(this);
+
+            //listening to dynamic configuration change
+            this.on('levelchange', function(level){
+                self.config.level = level;
+                renderer.renderChoices(self.id, self.dom, self.config);
+            });
+            $(this.dom).find('input').on('change', function() {
+                // Notify the response has changed
+                self.trigger('responseChange', [self.getResponse()]);
```

Comment:
**quibble:** not compliant with the IMS PCI standard, but this is acceptable with respect to the TAO flavor.